libsel4/tools/syscall_stub_gen.py

```diff
@@ -1069,6 +68,8 @@ def process_args():
     wsizegroup.add_argument("-c", "--cfile", dest="cfile",
                             help="Config file for Kbuild, used to get Word size.")
 
+    parser.add_argument("-l", "--lang", dest="lang", type=str.lower,
+                        choices=("c", "rust"), default="c")
```

Comment:
I wonder, can we not set a default here and force the caller to make a deliberate choice from now on? It might cause some inconvenience as some build workflow break, but the gain is less implicit behavior.

Reply:
I've removed the default value.
From what I see, in order to make it build properly, a change in `libsel4/CMakeLists.txt` was needed, I've modified it with `--lang=c`.